internal/mediainfo/language_test.go

```
@@ -0,0 +1,15 @@
+package mediainfo
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)